src/redux/actions/concepts/addBulkConcepts/index.js

```diff
@@ -57,15 +60,37 @@ export const previewConcept = id => (dispatch, getState) => {
   return dispatch(isSuccess(payload[0], PREVIEW_CONCEPT));
 };
 
-export const addConcept = (params, data, conceptName) => async (dispatch) => {
+
+export const recursivelyFetchConceptMappings = async (fromConceptCodes, levelsToCheck) => {
+  const startingConceptMappings = await api.mappings.fetchFromPublicSources(fromConceptCodes.join(','));
+  const mappingsList = [startingConceptMappings.data];
+  for (let i = 0; i < levelsToCheck; i += 1) {
```

Comment:
Is levelsToCheck different from LEVELS_TO_CHECK?

Reply:
They are the same. I thought it better to equate it to a constant as opposed to leaving it as a figure. Given the naming conventions used when assigning previous constants, I thought it better to maintain the standard

Reply:
Have your team mates seriously reviewed all this?

Reply:
I have shared this with the team and we need more clarity. `levelsToCheck` is a parameter, `LEVELS_TO_CHECK` represents the value that param takes. `LEVELS_TO_CHECK` currently is equal to the value `2`. Is the naming of the constant the issue or could you shed more light on what exactly we are missing?